plugins/modules/podman_image.py

```diff
@@ -175,6 +176,22 @@
             - docker-daemon
             - oci-archive
             - ostree
+    quadlet_dir:
+      description:
+        - Path to the directory to write quadlet file in.
```

Comment:
I think it would be better if we describe the default paths:
```suggestion
        - Path to the directory to write quadlet file in. By default, it will be set as `/etc/containers/systemd/` for root user, `~/.config/containers/systemd/` for non-root users.
```